gnpy/core/network.py

```diff
@@ -92,6 +90,9 @@ def split_fiber(network, fiber):
     network = add_egress_amplifier(network, prev_node)
     return network
 
+def select_edfa(ingress_span_loss):
+    return "std_medium_gain"
```

Comment:
`select_edfa` does not seem to be dependent on `ingress_span_loss`.

What are the other possibilities for return values?

Does this need to be a function right now?

Reply:
Easy,
work in progress here...

Reply:
@jeanluc-auge, this is a great addition to gnpy, thanks a lot for the hard work here! great stuff.